Makefile

```diff
@@ -123,7 +123,7 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 
 	# Optionally add/remove kernel options
 	if [ -f ../manage-config ]; then
-		../manage-config $(CONFIGURED_ARCH) $(CONFIGURED_PLATFORM)
+		../manage-config $(CONFIGURED_ARCH) $(CONFIGURED_PLATFORM) $(SECURE_UPGRADE_MODE) $(SECURE_UPGRADE_DEV_SIGNING_CERT)
```

Comment:
Should `SECURE_UPGRADE_MODE` be initialized somewhere in this file?

Reply:
its initiated by the slave container so didn't suffer an uninitialized issue, but I will add it anyway, cannot damage the code